lib/pakyow/command_line_interface.rb

```diff
@@ -0,0 +1,61 @@
+require "thor"
```

Comment:
I might just be missing it but do we still support `s` and `c` aliases? (e.g. `pakyow s` instead of `pakyow server`)